app/src/main/java/io/intelehealth/client/activities/patientSurveyActivity/PatientSurveyActivity.java

```diff
@@ -109,8 +113,17 @@ public void onClick(View v) {
         mSubmit.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                uploadSurvey();
-                endVisit();
+
+                if (rating != null && !TextUtils.isEmpty(rating)) {
+                    Log.d(TAG, "Rating is "+rating);
+                    uploadSurvey();
+                    endVisit();
+                }
+                else
+                {
+                    Toast.makeText(getApplicationContext(),"Please select a smiley before submitting",Toast.LENGTH_LONG).show();
```

Comment:
@prajwalmw - please use this as a String variable - "Please select a smiley before submitting" so that it can be translated into multiple languages

Reply:
Done! added it in the strings.xml file